packages/office-ui-fabric-react/src/components/Calendar/Calendar.scss

```diff
@@ -11,6 +11,7 @@ $Calendar-dayPicker-margin: 10px;
 
 .root {
   @include ms-normalize;
+  letter-spacing: -0.1px;
```

Comment:
It was in the zeplin styles for all the text, so that is why I added it.  But I will remove!